website/docs/guides/orchestration/running-dbt-cloud-jobs/1-types-of-jobs.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Organizing dbt Cloud jobs"
+id: "organizing-dbt-cloud-jobs"
+slug: 1-types-of-jobs
+description: Learn how to organize your dbt Cloud jobs by category
+displayText: The types of jobs in dbt Cloud
+hoverSnippet: Learn how to organize your dbt Cloud jobs by category.
+---
+
+dbt Cloud jobs perform two main functions:
+
+- Running models in production
+- Performing Continuous Integration Testing on models during development
+
+dbt Cloud jobs can optionally run source freshness checks and generate documentation as well.
+
+<callout>
+Make sure to leverage the dbt build command to avoid wasteful spend! dbt build enables you to run tests for each individual model, before its downstream dependencies start. When a test on a model fails, the dependences of that model are skipped. This enables you to avoid wasted spend on downstream node runs.
+</callout>
+
+### Running Models in Production
+
+#### Organizing models using tags
+
+Organizing your models in groups will allow you to create a successful orchestration strategy. Common classifications that users employ are:
+
+- Frequency: hourly, daily, weekly build schedules
+- Sensitivity: PII, GDPR, HIPAA
+- Domain: Marketing, Finance, Ops
+- Readiness: Staging, Intermediate, Marts / Bronze, Silver, Gold
+- dbt Configuration: Tables, Views, Incrementals, Snapshots
```

Comment:
Is this needed? Because it would be easier and more efficient to use model.materialisation rather than tags for scheduling purposes